test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetEndToEndTests/NuGetConsoleTestCase.cs

```diff
@@ -713,6 +712,84 @@ public async Task VerifyCacheFileInsideObjFolder(ProjectTemplate projectTemplate
             }
         }
 
+        [NuGetWpfTheory]
+        [MemberData(nameof(GetPackagesConfigTemplates))]
+        public async Task UpdateAllPCPackagesInPMC(ProjectTemplate projectTemplate)
+        {
+            EnsureVisualStudioHost();
+            using (var simpleTestPathContext = new SimpleTestPathContext())
+            {
+                // Arrange
+                var packageName1 = "UpdateAllPCPackage1";
```

Comment:
naming: I see no problem with calling it `packageA` or `packageB`.
It should be a bit more descriptive.

Reply:
Hi @nkolev92, thanks for your suggestion. In consideration of we need to login the machine to investigate test cases if they failed, we named it with the test case name. We thought we would know which test case is running when we see the package name.